### PR TITLE
Introduce OS module

### PR DESCRIPTION
**Pull request by gastaldi**
This contains OS related utilities (like detecting the Operating System it runs on and Process-related operations)

### CODE REVIEW

File: os/src/main/java/io/smallrye/common/os/ProcessRedirect.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    /**
+     * Get the "discard" process redirection target.
+     *
+     * @return the discarding redirection target
+     */
+    public static ProcessBuilder.Redirect discard() {
+        return ProcessBuilder.Redirect.to(new File(isWindows() ? "NUL" : "/dev/null"));
+    }
+
+    private static boolean isWindows() {
```
Comment (dmlloyd) [inline os/src/main/java/io/smallrye/common/os/ProcessRedirect.java:42]: I suppose this could delegate to OS.current()?

Comment (gastaldi) [reply os/src/main/java/io/smallrye/common/os/ProcessRedirect.java:42]: Good catch. Fixed

**Review by dmlloyd — APPROVED** [review]

**Comment by gastaldi** [comment]
Thanks! Any chance to have a release soon?

**Comment by radcortez** [comment]
@gastaldi done!

**Comment by gastaldi** [comment]
Thank you!

**Pull request merged**